docs/howto.rst

```diff
@@ -12,6 +12,18 @@ You can run the crontabber job that checks for security vulnerabilities locally:
    make dockerdependencycheck
 
 
+Connect to PostgreSQL Database
+==============================
+
+The docker container running the PostgreSQL database exposes itself on a
```

Comment:
Can you clarify that this is explicitly talking about the local development environment and not some docker container running elsewhere? In the "run security checks for dependencies", we use the word "locally". Something like that here would be super.

Reply:
Updated this to "The local development environment's PostgreSQL database exposes itself on a". Is that better?